modules/tvg_svginot/config.py

```diff
@@ -0,0 +1,7 @@
+def can_build(env, platform):
+    env.module_add_dependencies("svginot", ["freetype", "svg"])
```

Comment:
Shouldn't this match the module name, so `tvg_svginot`?

Though to be fair this module name is a bit confusing IMO. Maybe something like `opentype_svg`, or `svg_in_ot` and keeping the `tvg` relationship as implementation detail (like `svg`)?

Reply:
GCC on Linux won't found `extern` methods if it is compiled before text servers.

Reply:
Oh wow so it starts with `tvg` because `tvg > text_server`? That's pretty brittle :D

Reply:
For the same reason, all thridparty code is built as separate libs and appended after core, but won't work for this as it depends on thridparty code. I guess I'll just copy it to the both text servers, this will also help with GDExtension build.